darwin/DarwinProcess.c

```diff
@@ -194,7 +261,7 @@ static char* DarwinProcess_getCmdLine(const struct kinfo_proc* k, int* basenameO
    return retval;
 }
 
-void DarwinProcess_setFromKInfoProc(Process* proc, const struct kinfo_proc* ps, bool exists) {
+void DarwinProcess_setFromKInfoProc(DarwinProcess* proc, const struct kinfo_proc* ps, bool exists) {
```

Comment:
Canonically keep `Process* proc` and add a local variable `DarwinProcess* dp = (DarwinProcess*)proc;` directly at the start. Avoids having to indirect all the fields unnecessarily.

Reply:
Done.